joomla/joomla-cms
# [5.2] Email alt text on contact

Pull Request for Issue #37442 .

### Summary of Changes
The wrong language string was being referenced.
This had previously been fixed when using icons but was missed when using images

Comments about using alt titles and possibly invalid DT elements when using images instead of icons is beyond the scope of this PR. That is a bigger issue for the a11y team to consider

### Testing Instructions

- Go to Global Configuration -> Component -> Contacts -> Icon panel
- Set Settings drop-down list to Icons
- Select any picture for Custom Email Icon
- Create a contact, add every contact datas (name, address, email, etc.)
- Display the created contact using any kind of template (preferable Cassiopeia) which doesn't use template override for the file "components/com_contact/tmpl/contact/default_address.php"
- Check the alt text of the selected image of email


### Actual result BEFORE applying this Pull Request
The image used for the address image has a non translated value of alt="COM_CONTACT_EMAIL"

### Expected result AFTER applying this Pull Request
All the images have an ALT text description



### Link to documentations
Please select:
